hmda-data-publisher/src/main/scala/hmda/publisher/scheduler/LarScheduler.scala

```diff
@@ -59,6 +59,8 @@ class LarScheduler(publishingReporter: ActorRef[PublishingReporter.Command], sch
   val indexTractMap2021: Map[String, Census] = CensusRecords.indexedTract2021
   val indexTractMap2022: Map[String, Census] = CensusRecords.indexedTract2022
   val indexTractMap2023: Map[String, Census] = CensusRecords.indexedTract2023
+  val indexTractMap2024: Map[String, Census] = CensusRecords.indexedTract2024
+  val indexTractMap2025: Map[String, Census] = CensusRecords.indexedTract2024
```

Comment:
Should this be `indexedTract2025`?

Reply:
negative, 2025 census isn't available. We usually just rename 2024 to 2025 until it shows up next year but we're trying some thing new this go around and making it clear we're using 2024 census for 2025 references.

Reply:
negative since this is going to be referencing 2024 census until the actual 2025 is ready